user-interface/src/lib/components/uswds/todo.txt

```diff
@@ -0,0 +1,9 @@
+Missing USWDS Form Components:
+  1. Combo box
+  3. File input
```

Comment:
Do we need this file??

Reply:
I suppose not.  If not, then I'll save the file somewhere.  I think it's good to have a record of what isn't covered.  There was a group that was forming at Flexion to create a standard react and vue library for USWDS.  I imagine we could contribute our uswds components to it.